### PR TITLE
Add some more preliminary docs

### PR DESCRIPTION
**Pull request by shilgapira**
## Description
- Add some more preliminary docs

## Must
- [X] Tests
- [X] Documentation (if applicable)

### CODE REVIEW

File: docs/raw/applications/applications.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ oidc_applications
 
 - Type: `list` of `applications.OIDC`
```
Comment (ruvenzx) [inline docs/raw/applications/applications.md:10]: Please exclude the whole file from the PR, I have a much more descriptive information waiting on a different PR.

File: docs/raw/applications/oidc.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ id
 
 - Type: `string`
```
Comment (ruvenzx) [inline docs/raw/applications/oidc.md:10]: Please exclude the whole file from the PR, I have a much more descriptive information waiting on a different PR.

File: docs/raw/applications/saml.md [hunk 3]
```diff
@@ -9,7 +9,7 @@ id
 
 - Type: `string`
```
Comment (ruvenzx) [inline docs/raw/applications/saml.md:11]: Please exclude the whole file from the PR, I have a much more descriptive information waiting on a different PR.

File: docs/raw/attributes/attributes.md [hunk 4]
```diff
@@ -9,7 +9,7 @@ tenant
 
 - Type: `list` of `attributes.TenantAttribute` 
 
-// description for tenant
+Custom attributes to store additional details about your tenants.
```
Comment (ruvenzx) [inline docs/raw/attributes/attributes.md:12]: Please exclude the whole file from the PR, I have a much more descriptive information waiting on a different PR.

File: docs/raw/project.md [hunk 5]
```diff
@@ -18,7 +18,8 @@ environment
 
 - Type: `string`
```
Comment (ruvenzx) [inline docs/raw/project.md:19]: Please exclude the whole file from the PR, I have a much more descriptive information waiting on a different PR.

**Review by ruvenzx — CHANGES_REQUESTED** [review]

**Comment by shilgapira** [comment]
@ruvenzx Done

**Pull request merged**